jetson/nucleo_bridge/LCMHandler.cpp

```diff
@@ -64,11 +64,12 @@ void LCMHandler::InternalHandler::ra_closed_loop_cmd(LCM_INPUT, const ArmPositio
     ra_pos_data();
 }
 
-void LCMHandler::InternalHandler::sa_closed_loop_cmd(LCM_INPUT, const SAClosedLoopCmd *msg)
+void LCMHandler::InternalHandler::sa_closed_loop_cmd(LCM_INPUT, const SAPosition *msg)
```

Comment:
was there a specific reason from changing this from SA/RAClosedLoopCmd to SA/RAPosition? @CameronTressler @tabiosg

Reply:
Can just reuse an LCM (Already being used for reporting quadrature values). They both mean the try to mean the same thing anyway.

Reply:
gotcha